### DISCUSSION TITLE
Cambio_de_3er_integrante_del_grupo_trabajo_final

### DISCUSSION THREAD

**Discussion by solorzanob** [rramosp/ai4eng.v1#147] [UDEA Students]
Hola, buenos dias, un compañero cancelo la materia, por lo cual nos quedamos sin 1 integrante.
procedimos a darle el espacio a otro compañero, y volvi a llenar el formulario de inscripcion del equipo. La pregunta es, estuvo bien volver a llenar el formulario? o se debia hacer de alguna otra manera?

**Comment by rramosp** [comment]
hola solorzanob, así está perfecto. muchas gracias

**Comment by rramosp** [comment]
hola solorzanob, así está perfecto. muchas gracias

On Mon, Jun 27, 2022 at 12:05 PM solorzanob ***@***.***>
wrote:

> Hola, buenos dias, un compañero cancelo la materia, por lo cual nos
> quedamos sin 1 integrante.
> procedimos a darle el espacio a otro compañero, y volvi a llenar el
> formulario de inscripcion del equipo. La pregunta es, estuvo bien volver a
> llenar el formulario? o se debia hacer de alguna otra manera?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/147>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIKWSJ5QPD7REE3L3V3VRHNMVANCNFSM5Z6627BA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
